packages/js/src/Modules/Verto/util/helpers.ts

```diff
@@ -130,3 +131,17 @@ export const isValidOptions = ({
   const isToken = login_token;
   return Boolean(isLogin || isToken);
 };
+
+export const getGatewayState = (msg: IMessageRPC): GatewayStateType | '' => {
+  const hasStateResult: GatewayStateType | '' =
+    msg && msg.result && msg.result.params && msg.result.params.state
+      ? msg.result.params.state
+      : '';
```

Comment:
This statement is quite confusing. Could be better to use standard `if` condition here?

Reply:
Done!

```
export const getGatewayState = (msg: IMessageRPC): GatewayStateType | '' => {
  let stateResult: GatewayStateType | '' = '';
  let stateParam: GatewayStateType | '' = '';

  if (msg?.result?.params?.state) {
    stateResult = msg?.result?.params?.state;
  }

  if (msg?.params?.state) {
    stateParam = msg?.params?.state;
  }

  const gateWayState = stateResult || stateParam;

  return gateWayState;
};

```